Shared/InternalsVisibleTo.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("WPFClient")]
```

Comment:
Keep in mind, this at the moment is only needed because we don't have DI container setups yet. In the future this will have to be removed. The Communicator class should be internal, and the clients will only see the ICommunicator interface.